src/layout/mod.rs

```diff
@@ -0,0 +1,19 @@
+/**
+ * This file is part of Akira.
+ *
+ * Copyright (c) 2024 Alessandro Castellani
+ *
+ * Akira is free software: you can redistribute it and/or modify it under the
+ * terms of the GNU General Public License as  published by the Free Software
+ * Foundation, either version 3 of the License, or (at your option) any later version.
+ *
+ * Akira is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * Akira. If not, see <https://www.gnu.org/ licenses/>.
+ */
+pub mod left_sidebar;
+
+pub mod sidebars {}
```

Comment:
Seems unused

Reply:
Yes, thank you, I was experimenting with Rust modules and I need to clean this up